packages/eslint-config-typescript/index.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  extends: ["@madebywild/eslint-config-base"],
+  parser: "@typescript-eslint/parser",
```

Comment:
I think this belongs into the override as well. See https://github.com/facebook/create-react-app/blob/main/packages/eslint-config-react-app/index.js for an implementation example.